assets/src/dashboard/app/views/exploreTemplates/header/index.js

```diff
@@ -105,14 +104,40 @@ function Header({ filter, totalTemplates, search, templates, sort, view }) {
             onChange={onNewColorSelected}
           />
         </HeadingDropdownsContainer>
+      );
+    }
+    return null;
+  }, [
+    enableInProgressTemplateActions,
+    selectedCategories,
+    clearAllCategories,
+    onNewCategorySelected,
+    selectedColors,
+    clearAllColors,
+    onNewColorSelected,
+  ]);
+
+  return (
+    <Layout.Squishable>
+      <PageHeading
+        centerContent
+        defaultTitle={__('Templates', 'web-stories')}
+        searchPlaceholder={__('Search Templates', 'web-stories')}
+        stories={templates}
+        handleTypeaheadChange={
+          enableInProgressTemplateActions && search.setKeyword
```

Comment:
When enableInProgressTemplateActions is false the && operator will short circuit and return false which is incompatible with the propType.

(Look at the comment at line 140 to see the similar problem and solution)